### PR TITLE
feat(Session Replay): Maximum duration for a session replay

### PR DESCRIPTION
**Pull request by brustolin**
A session can't be longer than 60 minutes.

_#skip-changelog_

### CODE REVIEW

File: Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift [hunk 1]
```diff
@@ -48,6 +48,11 @@ public class SentryReplayOptions: NSObject {
      */
     let sessionSegmentDuration = TimeInterval(5)
     
+    /**
+     * The maximum duration of a replay session.
+     */
+    let maximumDuration = TimeInterval(3_600)
```
Comment (philipphofmann) [inline Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift:54]: `l`: Really 60 minutes? How can this be so high?

**Review by philipphofmann — APPROVED** [review]
LGTM

File: Sources/Sentry/SentrySessionReplay.m [hunk 2]
```diff
@@ -55,9 +57,10 @@ - (instancetype)initWithSettings:(SentryReplayOptions *)replayOptions
         _sentryRandom = random;
         _screenshotProvider = screenshotProvider;
         _displayLink = displayLinkWrapper;
-        _isRunning = false;
+        _isRunning = NO;
```
[the remark this answers is not in the document]
Comment (brustolin) [reply Sources/Sentry/SentrySessionReplay.m:60]: 😂  Im going from swift to objc in a daily basis, at this point its muscle memory

File: Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift (hunk 1 above)
Comment (brustolin) [reply Sources/Swift/Integrations/SessionReplay/SentryReplayOptions.swift:54]: @bruno-garcia ??

**Pull request merged**